cli/commands/benchmark-web-vitals.mjs

```diff
@@ -338,7 +357,7 @@ export async function handler( opt ) {
 	const params = getParamsFromOptions( opt );
 	const results = [];
 
-	const browser = await puppeteer.launch( { headless: 'new' } );
+	const browser = await puppeteer.launch( { headless: true } );
```

Comment:
This change was caught by TypeScript since Puppeteer the new headless mode is now the default.